mkdocs/mkdocs.yml

```diff
@@ -185,9 +199,4 @@ nav:
     - Analytics: contributing/analytics.md
     - Tasks: contributing/tasks.md
   - Roadmap: roadmap.md
-  - News: news.md
-
-# Google Analytics
-google_analytics:
```

Comment:
Did we intend to remove google_analytics directive?

Reply:
This method of adding Google Analytics is deprecated. I've added an updated version.